src/modules/views/Drive/DriveFolderView.jsx

```diff
@@ -156,7 +157,9 @@ const DriveFolderView = () => {
     canMove: true,
     isPublic: false,
     allLoaded,
-    showAlert
+    showAlert,
```

Comment:
This props should also be added inside `PublicFolderView`, `RecentView` and `SharingsFolderView` because they are using also the action `trash`